docs/02-app/02-api-reference/04-functions/image-response.mdx

```diff
@@ -35,3 +35,10 @@ new ImageResponse(
 ## Supported CSS Properties
 
 Please refer to [Satori’s documentation](https://github.com/vercel/satori#css) for a list of supported HTML and CSS features.
+
+## Version History
+
+| Version   | Changes                                             |
+| --------- | --------------------------------------------------- |
+| `v13.3.0` | `ImageReponse` can be imported from `next/server`.  |
+| `v13.0.0` | `ImageReponse` introduced via `@vercel/og` package. |
```

Comment:
I think you can remove the 13.0.0 part here. You could use `@vercel/og` with any version of any framework.

The 13.3.0 is when it was introduced to Next.js so that is the only relevant line.